All Core Devs Meetings/meeting 94.md

```diff
@@ -0,0 +1,437 @@
+  
+**Hudson** - great. I do want to ask I'm not seeing in the decisions made section in the notes anything about us definitely spinning up testnets for prog pow or spinning up Ropstein or anything specific like that. Did we in your recollection, was there like a specific tests that we talked about spinning up for it, or was it Ropstein or what?
+  
+**Greg** - No doubt in two weeks, those are the sorts of details we can discuss if we decide to move forward.
+  
+**Hudson** - Ok, I would say, like I disagree that it is a decision made that we are spinning up tests. That's for that, because from before the meeting, looking through notes and meeting eighty two and eighty one, I, I'll need to look through the videos, but I didn't see any definitive consensus around that. And you even I think you cited a few people who said that they you thought they had consensus around it, but I don't remember them publicly saying anything about it.
+  
+**Greg** - It's such a mess, the consensus was so rough that we need to revisit that. I know on Twitter vitalik and and on the EIP on Twitter channels, Vitalik and I, you know, sort of agreed that the rough consensus in 82 was was bandz compromise, which lays it out that way. And the current EIP lays it out that way. But it's more detailed decisions we need to make on on what test set to bring it up on.
+  
+**Martin** - And I'd also like to kind of agree that we did talk about testnets, but I also I really think that's something we need consensus on and it's just a matter of standing up. If that's not those who wants to join can join those who don't. Don't.
+  
+**Hudson** - Oh, yeah. No, it's not I don't mean that we have to have consensus to spin up a test that we would for Ropstein, probably more so than just a random test that. Would you agree?
+  
+**James** - And something I have question I have for the groups we've talked earlier about how Rollston is at the point it's worth perhaps restarting. Is that still something that we think about or we still think is the case, or am I wrong that.
+  
+**Martin** - Yeah, before we go into that, yeah, I just want to answer Hudson's question. So the funny thing is, if we are to if we were to actually do a profile on Ropstein, is that I don't think it would cause any problems because I know first cinching would start with a Herbertson and would immediately see like what strange difficulties is this and it'll get rejected. So it's a lot easier than, for example, if we did the hard fork with an evm change, that's not immediately noticeable when you first think. So there's a difference there.
+  
+**Hudson** - Ok, cool, and then, James, you were asking everybody about rebooting Robstein or not rebooting Robstein but retiring Robstein, I guess you would say. Right?
```

Comment:
Ok, cool, and then, James, you were asking everybody about rebooting Ropsten or not rebooting Robstein but retiring Ropsten, I guess you would say. Right?